cmd/kube-apiserver/app/server.go

```diff
@@ -269,25 +269,25 @@ func CreateNodeDialer(s *options.ServerRunOptions) (tunneler.Tunneler, *http.Tra
 }
 
 // CreateKubeAPIServerConfig creates all the resources for running the API server, but runs none of them
-func CreateKubeAPIServerConfig(s *options.ServerRunOptions, nodeTunneler tunneler.Tunneler, proxyTransport *http.Transport) (*master.Config, informers.SharedInformerFactory, clientgoinformers.SharedInformerFactory, *kubeserver.InsecureServingInfo, aggregatorapiserver.ServiceResolver, error) {
+func CreateKubeAPIServerConfig(s *options.ServerRunOptions, nodeTunneler tunneler.Tunneler, proxyTransport *http.Transport) (*master.Config, informers.SharedInformerFactory, clientgoinformers.SharedInformerFactory, *kubeserver.InsecureServingInfo, aggregatorapiserver.ServiceResolver, []admission.PluginInitializer, error) {
```

Comment:
Break the line?

Reply:
return a new param admission pluginInitializers

Reply:
I meant that if you could break this long line into multiple lines, like what you did [here](https://github.com/kubernetes/kubernetes/pull/61404/files/85e69c240c449762d50b0e2fa9687f7efd1bde84#diff-13674dbb1dbd04c901cd058d402e71a3R33).

Reply:
ic, forgive my poor English